alembic/versions/e511d62f38b7_channels_table_constraint_keys.py

```diff
@@ -0,0 +1,45 @@
+"""channels table constraint keys
+
+Revision ID: e511d62f38b7
+Revises: 8efbde3ecaeb
+Create Date: 2019-12-06 10:33:42.532924
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e511d62f38b7'
+down_revision = '8efbde3ecaeb'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("channels") as batch_op:
+        batch_op.create_foreign_key("fk_channels_bouquets",
+                                    "bouquets", ["bouquet_id"], ["id"],
+                                    ondelete='CASCADE')
+    with op.batch_alter_table("channels") as batch_op:
+        batch_op.create_unique_constraint("uk_calendar_id",
+                                          ["calendar_id"])
+
+
+def downgrade():
+    op.drop_table('channels')
+    op.create_table('channels', sa.Column('date_created', sa.DateTime(),
```

Comment:
Was this file auto-generated? Why did you have to drop and create the table? Couldn't you just remove the constraints?

Reply:
No, it wasn't auto-generated. I used this way because it was impossible to drop the constraint using `op` at the time, but now with the use of `batch` I will just drop these constraints.